README.md

```diff
@@ -1,7 +1,21 @@
 # Poietic Flows
 
 Systems Dynamics Modelling and simulation toolkit based on the
-[Stock and Flow](https://en.wikipedia.org/wiki/Stock_and_flow) methodology.
+[Stock and Flow](https://en.wikipedia.org/wiki/Stock_and_flow) methodology .
```

Comment:
Describing what this repo does with a link to the wikipedia description is a really nice start.  I read so many repos and have to get deep into the README before I even have an idea about the problem it is trying to solve.

That said, you have 3 links to the same wikipedia article in the README